### PR TITLE
Style Fix

### PR DESCRIPTION
**Pull request by kemaswill**
Style fix for keras/layers/advanced_activations.py

### CODE REVIEW

File: examples/deep_dream.py [hunk 1]
```diff
@@ -140,7 +140,7 @@ def continuity_loss(x):
 grads = K.gradients(loss, dream)
 
 outputs = [loss]
-if type(grads) in {list, tuple}:
+if isinstance(grads, list) or isinstance(grads, tuple):
```
Comment (fchollet) [inline examples/deep_dream.py]: You can use `isinstance` with tuples of types.

File: examples/neural_doodle.py [hunk 2]
```diff
@@ -301,7 +301,7 @@ def total_variation_loss(x):
 
 # Evaluator class for computing efficiency
 outputs = [loss]
-if type(loss_grads) in {list, tuple}:
+if isinstance(loss_grads, list) or isinstance(loss_grads, tuple):
```
Comment (fchollet) [inline examples/neural_doodle.py]: Same - use a tuple of types.

File: examples/neural_style_transfer.py [hunk 3]
```diff
@@ -208,7 +208,7 @@ def total_variation_loss(x):
 grads = K.gradients(loss, combination_image)
 
 outputs = [loss]
-if type(grads) in {list, tuple}:
+if isinstance(grads, list) or isinstance(grads, tuple):
```
Comment (fchollet) [inline examples/neural_style_transfer.py]: Same

File: keras/layers/advanced_activations.py [hunk 4]
```diff
@@ -68,7 +68,7 @@ def __init__(self, init='zero', weights=None, shared_axes=None, **kwargs):
         self.supports_masking = True
         self.init = initializations.get(init)
         self.initial_weights = weights
-        if type(shared_axes) is not list and type(shared_axes) is not tuple:
+        if not isinstance(shared_axes, list) and not isinstance(shared_axes, tuple):
```
Comment (fchollet) [inline keras/layers/advanced_activations.py]: Same

File: keras/layers/advanced_activations.py [hunk 5]
```diff
@@ -171,7 +171,7 @@ def __init__(self, alpha_init=0.2, beta_init=5.0,
         self.alpha_init = K.cast_to_floatx(alpha_init)
         self.beta_init = K.cast_to_floatx(beta_init)
         self.initial_weights = weights
-        if type(shared_axes) is not list and type(shared_axes) is not tuple:
+        if not isinstance(shared_axes, list) and not isinstance(shared_axes, tuple):
```
Comment (fchollet) [inline keras/layers/advanced_activations.py]: Same

File: keras/layers/advanced_activations.py [hunk 6]
```diff
@@ -276,7 +276,7 @@ def __init__(self, t_left_init='zero', a_left_init='glorot_uniform',
         self.a_left_init = a_left_init
         self.t_right_init = t_right_init
         self.a_right_init = a_right_init
-        if type(shared_axes) is not list and type(shared_axes) is not tuple:
+        if not isinstance(shared_axes, list) and not isinstance(shared_axes, tuple):
```
Comment (fchollet) [inline keras/layers/advanced_activations.py]: Same

File: keras/utils/layer_utils.py [hunk 7]
```diff
@@ -126,7 +126,7 @@ def count_total_params(layers, layer_set=None):
         if layer in layer_set:
             continue
         layer_set.add(layer)
-        if type(layer) in (Model, Sequential):
+        if isinstance(layer, Model) or isinstance(layer, Sequential):
```
Comment (fchollet) [inline keras/utils/layer_utils.py]: Same

**Comment by kemaswill** [comment]
Done.

**Comment by fchollet** [comment]
Thanks!

**Pull request merged**